engine/core/src/main/java/org/datacleaner/util/http/CASMonitorHttpClient.java

```diff
@@ -65,6 +65,10 @@
 
     private static final Logger logger = LoggerFactory.getLogger(CASMonitorHttpClient.class);
 
+    private static final String CDI_TICKET_HEADER = "CDI-ticket";
+    private static final String CDI_SERVICE_URL_HEADER = "CDI-serviceUrl";
+    private static final String CDI_USERID = "CDI-userId";
```

Comment:
This is not proper CAS compliant. This is _HI specific_ CAS code. I don't think that's the proper way to do it.